centreon/www/class/centreonLogAction.class.php

```diff
@@ -212,13 +212,17 @@ public function getHostName($host_id)
             return $info['host_name'];
         }
 
-        $query = "SELECT object_id, object_name FROM log_action WHERE object_type = 'service' AND object_id = $host_id";
+        $query = "SELECT object_id, object_name FROM log_action WHERE object_type = 'service' AND object_id = " . $host_id;
         $DBRESULT2 = $pearDBO->query($query);
         $info = $DBRESULT2->fetchRow();
         if (isset($info['object_name'])) {
             return $info['object_name'];
         }
-        return -1;
+        
+        $query = "SELECT name FROM hosts WHERE host_id = " . $host_id;
```

Comment:
Prepare statement please
```suggestion
        $query = "SELECT name FROM hosts WHERE host_id = " . $host_id;
```

Reply:
You are right. But to me, this is weird to ask that :
1. Either we fix all the remaining holes in the function.
2. Either we stay with the same legacy code style only fixing what we want.

But being between the two is not consistent to me.